wait/host_port.go

```diff
@@ -57,8 +57,23 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 
 	var waitInterval = 100 * time.Millisecond
 
+	internalPort := hp.Port
+	if internalPort == "" {
+		var ports nat.PortMap
+		ports, err = target.Ports(ctx)
+		if err != nil {
+			return
+		}
+		if len(ports) > 0 {
```

Comment:
Is it possible that `target.Ports(ctx)` retrieves an empty array, so this condition is never met, therefore internalPort is still empty?

Reply:
Fixed, return error of internalPort is empty.